.github/pipelines.sh

```diff
@@ -1,11 +1,13 @@
 #!/bin/bash
-set -ex
+set -e
 YQ="${YQ:-yq}"
 
 for i in $(ls config); do
     filename=$(basename -s .yaml $i)
-    for arch in $($YQ r config/"$i" -p "p" 'arches.[*]' | cut -d "." -f 2); do
-      $YQ r config/"$i" "arches[$arch]" | gomplate --left-delim "{{{" --right-delim "}}}" -V --datasource config="stdin:/?type=application/yaml" --file build.yaml.gomplate --out workflows/build-"$filename"-"$arch".yaml
-      sed -i '/^[[:space:]]*$/d' workflows/build-"$filename"-"$arch".yaml
+    for flavor in $($YQ r config/"$i" -p "p" 'flavors.[*]' | cut -d "." -f 2); do
```

Comment:
A small nit here, we might start to handle this loops directly in the template instead as they are tending to grow :smile: , so we iterate directly from gomplate through all the arch/flavors instead of doing it in bash

Reply:
Problem is, if we do on the template then we get only 1 file, i.e. if we deal with flavors on the template side, we got again 1 workflow with all the flavors, while the point here is to separate those into different workflows and files for easy manage.

But I agree that this should be done differently somehow, maybe moving to yq4 or using jq to select the flavor+arch more precisely than having to parse the output like we do now